website/sidebars.js

```diff
@@ -31,61 +42,112 @@ const sidebarSettings = {
     },
     {
       type: "category",
-      label: "Get started with dbt",
+      label: "Set up dbt",
       collapsed: true,
       items: [
         {
           type: "category",
```

Comment:
does it make sense to include an "About setup" page ?

Reply:
Written and submitting for review!